packages/atomic/scripts/copy-dayjs-locales.js

```diff
@@ -0,0 +1,24 @@
+const util = require('util');
+const fs = require('fs');
+const rm = util.promisify(fs.rm);
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+async function copyDayjsLocales() {
+  const localesJSONData = await readFile('node_modules/dayjs/locale.json');
+  const localesData = JSON.parse(localesJSONData);
+  let fileContent =
+    'export const locales: Record<string, () => Promise<unknown>> = {';
```

Comment:
What would we gain (if anything) if there was a proper type instead of `unknown` here ?